static/src/javascripts/projects/common/modules/commercial/templates/acquisitions-epic-us-leading-with-climate.js

```diff
@@ -0,0 +1,53 @@
+// @flow
+import { appendToLastElement } from 'lib/array-utils';
+
+const gridImage = `
+    <div class="u-responsive-ratio" style="padding-bottom: 60.01%;">
```

Comment:
I took this directly from the HTML for the image at the top of this article, which is about the same size: https://www.theguardian.com/us-news/2018/nov/09/california-wildfire-paradise-chico-camp-fire

There are a bunch of abstractions around the grid stuff which provide all these different `srcset` images at different resolutions for the right screen size, but these are all serverside. So I thought I'd just copy-paste the whole lot to use them clientside.